mlir/include/mlir/Dialect/ArmSME/IR/ArmSMEOps.td

```diff
@@ -27,7 +27,7 @@ include "mlir/Interfaces/InferTypeOpInterface.td"
 
 class SMETileType<Type datatype, list<int> dims, string description>
   : ShapedContainerType<[datatype],
-      And<[IsVectorOfRankPred<[2]>, allDimsScalableVectorTypePred,
+      And<[IsVectorOfRankPred<[2]>, IsVectorTypeWithAllDimsScalablePred,
```

Comment:
Not required: The `IsVectorOfRankPred<[2]>` is redundant `dims` list always has two elements. 
```suggestion
      And<[IsVectorTypeWithAllDimsScalablePred,
```

Reply:
Ha, interesting! I will keep it as a useful bit of documentation.